lightning-block-sync/src/rpc.rs

```diff
@@ -68,9 +87,11 @@ impl RpcClient {
 
 		let error = &response["error"];
 		if !error.is_null() {
-			// TODO: Examine error code for a more precise std::io::ErrorKind.
```

Comment:
Looking into this a little more, I think the idea was that some errors may be considered transient while others are persistent. And this would affect the conversion done here: https://github.com/lightningdevkit/rust-lightning/blob/8311581fe110a9ee561a6fda6b55c78a02068d43/lightning-block-sync/src/convert.rs#L21-L29

This let's the user know if they should give-up or retry.

Reply:
Ah ok, that makes sense.  Well I can put the TODO back.  Seems like to actually impl the TODO we would need to map the error rpc error codes into buckets for persistent vs transient.